packages/runner/injection/multi-domain.js

```diff
@@ -16,7 +16,18 @@ const findCypress = () => {
     try {
       // If Cypress is defined and we haven't gotten a cross origin error we have found the correct bridge.
       if (frame.Cypress) {
-        return frame.Cypress
+        // If the ending $ is not included in the template string, it breaks transpilation
```

Comment:
```suggestion
        // If the ending $ is included in the template string, it breaks transpilation
```

It breaks it if it _is_ included, right? Does it still break if you escape the $ (e.g. `\$`)?

Reply:
My wording is confusing 🤪 .

This string breaks:
```
`(^|\\.)${ frame.location.host.replaceAll('.', '\\.') }$`
```

This string works: 
```
`(^|\\.)${ frame.location.host.replaceAll('.', '\\.') }` + '$'
```